content/backmatter.tex

```diff
@@ -1249,4 +1249,41 @@ \section{Version 1.1}
 %
 \end{itemize}
 
+\chapter{Errata}\label{sec:errata}
+
+The \openshmem specification may occasionally include errata that are
+discovered after the release of new versions.
+These errata can range from typographical mistakes to more significant
+technical inaccuracies that may affect the implementation or understanding of
+the \openshmem API semantics.
+This Errata section reports these issues with the goal to maintain the
+integrity and utility of the OpenSHMEM specification.
+
+The sections below document all known errata, their corresponding corrections,
+and the affected version of the OpenSHMEM specification.
+It serves as a historical record of the changes made and assists users and
+implementers with applying the necessary corrections.
+
+\section{Version 1.5}
+
+\begin{itemize}
+  \item Clarified that \FUNC{shmem\_test\_all} and
+     \FUNC{shmem\_test\_all\_vector} routines return 1 when the test set is empty
+        (\href{https://github.com/openshmem-org/specification/pull/466}{\#466}).
+  \item Clarified that \FUNC{shmem\_team\_split\_strided} and
+     \FUNC{shmem\_team\_split\_2d} return nonzero when the parent team is
+     \LibConstRef{SHMEM\_TEAM\_INVALID}
+        (\href{https://github.com/openshmem-org/specification/pull/461}{\#461}).
+  \item Corrected the \VAR{level} argument's recommended value in API notes for
+     \FUNC{shmem\_pcontrol} to indicate that the value should be greater than 2 to enable
+     profiling with profile library defined effects and additional arguments
+        (\href{https://github.com/openshmem-org/specification/pull/480}{\#480}).
+  \item Clarified that fence operations only guarantee ordering for operations
+     that are performed on the same context
+        (\href{https://github.com/openshmem-org/specification/pull/496}{\#496}).
+  \item Clarified that \VAR{source} and \VAR{dest} arrays must be the same
+      across \acp{PE} in \openshmem reductions
+        (\href{https://github.com/openshmem-org/specification/pull/490}{\#490}).
+\end{itemize}
+
```

Comment:
@davidozog I noticed that these are duplicated in the Change Log. Do you think we should break these out from there as errata?

Shmem_test_all: Missing in Changelog

[Team_Strided](https://github.com/kwaters4/specification/blob/1d6f40e2cd9b1c71754a3f45d4bc124b9065c5cd/content/backmatter.tex#L679C1-L679C2)

[Level](https://github.com/kwaters4/specification/blob/1d6f40e2cd9b1c71754a3f45d4bc124b9065c5cd/content/backmatter.tex#L706)

[Fence](https://github.com/kwaters4/specification/blob/1d6f40e2cd9b1c71754a3f45d4bc124b9065c5cd/content/backmatter.tex#L671)

Reductions buffers: Missing in Changelog